awspub/tests/test_context_validation.py

```diff
@@ -0,0 +1,43 @@
+import pytest
```

Comment:
I think these tests should be included in the already existing `test_context.py`